packages/vm/src/ibc_calls.rs

```diff
@@ -231,3 +231,118 @@ where
         MAX_LENGTH_IBC,
     )
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::calls::{call_handle, call_init};
+    use crate::testing::{mock_env, mock_info, mock_instance, MockApi, MockQuerier, MockStorage};
+    use cosmwasm_std::testing::{mock_ibc_channel, mock_ibc_packet_ack};
+    use cosmwasm_std::{Empty, IbcOrder};
+
+    static CONTRACT: &[u8] = include_bytes!("../testdata/ibc_reflect.wasm");
+    const IBC_VERSION: &str = "ibc-reflect";
+
+    #[test]
+    fn call_init_works() {
+        let mut instance = mock_instance(&CONTRACT, &[]);
+
+        // init
+        let info = mock_info("creator", &[]);
+        let msg = br#"{"reflect_code_id":77}"#;
+        call_init::<_, _, _, Empty>(&mut instance, &mock_env(), &info, msg)
+            .unwrap()
+            .unwrap();
+    }
```

Comment:
At this place we are not testing the contract but the calls and `call_init` is tested in the other file. So I think this test should be remioved.